doc/man5/flux-config-job-manager.rst

```diff
@@ -110,28 +110,31 @@ prolog
    supported for the ``[job-manager.prolog]`` table:
 
    command
-      (optional) An array of strings specifying the command to run. If
+      (optional, string) An array of strings specifying the command to run. If
       ``exec.imp`` is set, the the default command is ``["flux-imp",
       "run", "prolog"]``, otherwise it is an error if command is not set.
    per-rank
-      (optional) By default the job-manager prolog only runs ``command``
+      (optional, bool) By default the job-manager prolog only runs ``command``
       on rank 0. With ``per-rank=true``, the command will be run on each
       rank assigned to the job.
    timeout
-      (optional) A string value in Flux Standard Duration specifying a
+      (optional, string) A string value in Flux Standard Duration specifying a
       timeout for the prolog, after which it is terminated (and a job
       exception raised). The default prolog timeout is 30m. To disable
       the timeout use ``0`` or ``infinity``.
    kill-timeout
-      (optional) If a job exception is raised during the job prolog and
-      ``cancel-on-exception`` is true,  the prolog will be canceled by
-      sending it a SIGTERM signal. ``kill-timeout`` is the number of
-      seconds to wait until any nodes with prolog tasks that are still
-      active will be drained. The drain reason will include the string
-      "canceled then timed out". The default is 60.
+      (optional, float) If a the prolog times out, or a job exception is raised
+      during the job prolog and ``cancel-on-exception`` is true, the prolog
+      will be canceled by sending it a SIGTERM signal. ``kill-timeout``
+      is a floating point number of seconds to wait until any nodes with
+      prolog tasks that are still active will be drained. The drain reason
+      will include the string "timed out" if the prolog timeout was reached,
+      or "canceled then timed out" if the prolog was canceled after a job
+      exception then timed out. The default is 60.
    cancel-on-exception
-      (optional) A boolean indicating whether a fatal job exception raised
-      while the prolog is active terminates the prolog. The default is true.
+      (optional, bool) A boolean indicating whether a fatal job exception is
+      raised while the prolog is active terminates the prolog. The default
```

Comment:
both here and in epilog, first sentence seems off to me with the "while the prolog is active terminates the prolog".  Perhaps reorder the sentence,  "A boolean indicating if a prolog should be terminated if a fatal job exception is raised while the prolog is active."?

Reply:
Oops, this was an edit issue - two versions of the sentence remain grafted together here. Anyway, I like your suggestion so let's use that.